### PR TITLE
release: v1.37.0

### PR DESCRIPTION
**Pull request by MarlonGamez**
Release PR for v1.37.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,81 @@
...
+
+Highlights:
+* Allow and deny list support for schema field overwriting [#7056](https://github.com/GoogleContainerTools/skaffold/pull/7056)
+* Cross-platform support for kaniko builder [#7142](https://github.com/GoogleContainerTools/skaffold/pull/7142)
+* Cross-platform support for GCB build env [#7134](https://github.com/GoogleContainerTools/skaffold/pull/7134)
+* Cross-platform support for jib builder [#7110](https://github.com/GoogleContainerTools/skaffold/pull/7110)
+
+New Features and Additions:
+* feat: add basic metrics for resource selector usage [#7192](https://github.com/GoogleContainerTools/skaffold/pull/7192)
+* feat: Allow Confluent-for-Kubernetes types to be transformed [#7179](https://github.com/GoogleContainerTools/skaffold/pull/7179)
```
Comment (tejal29) [inline CHANGELOG.md:30]:
```suggestion
* feat: Allow [Confluent-for-Kubernetes](https://docs.confluent.io/operator/current/overview.html?_ga=2.137619952.1165154847.1647462431-157638067.1647462431&_gac=1.194052959.1647462431.CjwKCAjwlcaRBhBYEiwAK341jVVmaAbOOVWBcuYdjPp3PtwfEbiz48zXgui9hhn5stfT-G9JULvLuBoCZOIQAvD_BwE) types to be transformed [#7179](https://github.com/GoogleContainerTools/skaffold/pull/7179)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,81 @@
...
+**Windows**
+https://storage.googleapis.com/skaffold/releases/v1.37.0/skaffold-windows-amd64.exe
+
+**Docker image**
+`gcr.io/k8s-skaffold/skaffold:v1.37.0`
+
+Note: This release comes with a new config version, `v2beta28`. To upgrade your skaffold.yaml, use `skaffold fix`. If you choose not to upgrade, skaffold will auto-upgrade as best as it can.
+
+Highlights:
+* Allow and deny list support for schema field overwriting [#7056](https://github.com/GoogleContainerTools/skaffold/pull/7056)
```
Comment (tejal29) [inline CHANGELOG.md:23]:
```suggestion
* New Resource selector to allow or deny transforming specific resources in deploy stage [#7056](https://github.com/GoogleContainerTools/skaffold/pull/7056)
```

**Review by tejal29 — COMMENTED** [review]
Are we missing contributors list?

**Comment by MarlonGamez** [comment]
@tejal29 thanks for pointing out, I totally forgot to copy that over 🤦‍♂️

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,101 @@
...
+- Christopher Bartz
+- Dan Williams
+- Gaurav
+- Halvard Skogsrud
+- Marlon Gamez
+- Michael Mohamed
+- Riccardo Carlesso
+- Savas Ersin
+- Steven Powell
+- Tejal Desai
```
Comment (tejal29) [inline CHANGELOG.md:95]: always happy to see my name.

**Review by tejal29 — APPROVED** [review]

**Pull request merged**